client/offchain/src/api/http.rs

```diff
@@ -820,6 +820,9 @@ mod tests {
 		};
 	}
 
+	// Todo: Enable this test when polkadot-v0.9.3 is merged
+	// https://github.com/paritytech/substrate/tree/polkadot-v0.9.3
+	#[ignore]
 	#[test]
 	fn request_write_body_invalid_call() {
```

Comment:
[This test is flaky](https://github.com/paritytech/substrate/issues/5380). It will be [fixed](https://github.com/paritytech/substrate/pull/6038) in 0.9.3.